orangecontrib/timeseries/widgets/owtimeslice.py

```diff
@@ -206,46 +208,64 @@ def datetime_edited(dt_edit):
             lambda: date_to.calendarWidget().repaint()
         )
 
-        hbox.layout().addStretch(100)
-        hbox.layout().addWidget(date_from)
-        hbox.layout().addWidget(QLabel(' – '))
-        hbox.layout().addWidget(date_to)
-        hbox.layout().addStretch(100)
+        dtBox.layout().addStretch(100)
+        dtBox.layout().addWidget(date_from)
+        dtBox.layout().addWidget(QLabel(' – '))
+        dtBox.layout().addWidget(date_to)
+        dtBox.layout().addStretch(100)
 
-        vbox = gui.vBox(self.controlArea, 'Step / Play Through')
-        gui.checkBox(vbox, self, 'loop_playback',
+        hCenterBox = gui.hBox(self.controlArea)
+        gui.rubber(hCenterBox)
+        vControlsBox = gui.vBox(hCenterBox)
+
+        stepThroughBox = gui.vBox(vControlsBox, 'Step/Play Through')
+        gui.rubber(stepThroughBox)
+        gui.checkBox(stepThroughBox, self, 'loop_playback',
                      label='Loop playback')
-        hbox = gui.hBox(vbox)
-        gui.checkBox(hbox, self, 'custom_step_size',
-                     label='Custom step size:',
+        customStepBox = gui.hBox(stepThroughBox)
+        gui.checkBox(customStepBox, self, 'custom_step_size',
+                     label='Custom step size: ',
                      toolTip='If not chosen, the active interval moves forward '
                              '(backward), stepping in increments of its own size.')
-        self.stepsize_combobox = gui.comboBox(hbox, self, 'step_size',
+        self.stepsize_combobox = gui.comboBox(customStepBox, self, 'step_size',
                                               items=tuple(self.STEP_SIZES.keys()),
                                               sendSelectedValue=True)
-        hbox = gui.hBox(vbox)
-        self.step_backward = gui.button(hbox, self, '⏮',
+        playBox = gui.hBox(stepThroughBox)
+        gui.rubber(playBox)
+        gui.rubber(stepThroughBox)
+
+        self._play_icon = QIcon(join(dirname(__file__), 'icons', 'TimeSlice_play.svg'))
```

Comment:
These files don't exist (yet?). We should add the icons before merging.

Reply:
Oops, pushed them.